distribution/ddf-common/src/main/resources/etc/ws-security/server/encryption.properties

```diff
@@ -5,4 +5,4 @@ org.apache.ws.security.crypto.merlin.keystore.password=${javax.net.ssl.keyStoreP
 org.apache.ws.security.crypto.merlin.keystore.private.password=${javax.net.ssl.keyStorePassword}
 org.apache.ws.security.crypto.merlin.keystore.alias=${org.codice.ddf.system.hostname}
 org.apache.ws.security.crypto.merlin.keystore.file=${javax.net.ssl.keyStore}
-#org.apache.ws.security.crypto.merlin.x509crl.file=etc/certs/demoCA/crl/crl.pem
```

Comment:
Then all iTests fail. As long as that is commented out we can't invoke `migration:export` on command line.